.github/workflows/production.yml

```diff
@@ -97,7 +97,7 @@ jobs:
             - name: Verify nginx image
               run: |
                   set -e
-                  #docker run --rm -it --entrypoint sh ${{ secrets.DOCKHUB_ORGANISATION }}/deriv-com:latest -c "nginx -t"
+                  docker run --rm --entrypoint sh ${{ secrets.DOCKHUB_ORGANISATION }}/deriv-com:latest -c "nginx -t"
```

Comment:
we can omit `entrypoint` and `-c` parameter as `regentmarkets/deriv-com` is based on nginx:alpine image which doesn't support this